ElevenLabs-DotNet/TextToSpeech/TextToSpeechEndpoint.cs

```diff
@@ -34,23 +34,6 @@ public sealed class TextToSpeechEndpoint : BaseEndPoint
         /// <returns>Downloaded clip path.</returns>
         public async Task<string> TextToSpeechAsync(string text, Voice voice, VoiceSettings voiceSettings = null, Model model = null, string saveDirectory = null, bool deleteCachedFile = false, CancellationToken cancellationToken = default)
         {
-            if (text.Length > 5000)
```

Comment:
Let's keep the parameter checks

Reply:
we're calling `TextToSpeechStreamAsync` down below; all the validation went there because it is reused by the byte[] version